activerecord/lib/active_record/associations.rb

```diff
@@ -504,7 +504,7 @@ def association_instance_set(name, association)
     #
     # == Customizing the query
     #
-    # \Associations are built from <tt>Relation</tt>s, and you can use the Relation syntax
+    # \Associations are built from <tt>Relations</tt>, and you can use the Relation syntax
```

Comment:
I agree that it looks odd on the API site but what do you think about `Associations are built from +Relation+ objects` ?